Sdk/README.md

```diff
@@ -0,0 +1,318 @@
+> :warning: **This Interface should be implemented by DeBot Engine**
+# System Interface
+
+**Status**: Proposed
+
+| Name            | ID                                                                |
+| :-------------- | :---------------------------------------------------------------- |
+| Sdk             | 8fc6454f90072c9f1f6d3313ae1608f64f4a0660c6ae9f42c68b6a79e2a1bc4b  |
+
+
+## Description
+
+Builtin Interface that covers subset of TON SDK functions. Implemented by DEngine.
+
+## Functions
+
+Since DeBot is a smart contract then all functions work asyncronously by design. It means that they don't return anything despite the fact that all have `returns` value in their specification. Result callback will be called later and it must have arguments defined in `returns` section.
+
+### Accounts
+
+`getBalance` - get account balance. 
+
+arguments: 
+
+	answerId: uint32 - function id of result callback.
+	addr    : address - address of contract.
+
+returns: 
+
+	nanotokens: uint128 - account balance in nanotokens.
+
+`getAccountType` - get account type. 
+
+arguments: 
+
+	answerId: uint32 - function id of result callback.
+	addr    : address - address of contract.
+
+returns: 
+
+	acc_type: int8 - account type. Valid values are -1 (unexist), 0 (uninit), 1 (active), 2 (frozen).
+
+`getAccountCodeHash` - get account code hash.
+
+arguments: 
+
+	answerId: uint32 - function id of result callback.
+	addr    : address - smart contract address.
+
+returns: 
+
+	code_hash: uint256 - account code hash.
+
+### Encryption
+
+`encrypt` - encrypt data with defined encryption box.
+
+arguments:
+
+    answerId: uint32 - function id of result callback.
+    boxHandle: uint32 - encryption box handle. Remark: it can be received through `EncryptionBoxInput` interface.
+
+returns:
+
+    result: uint32 - result code of operation. 0 if succeeded, otherwise SDK error code.
+    encrypted: bytes - encrypted data bytes.
+
+`decrypt` - decrypt data with defined encryption box.
+
+arguments:
+
+    answerId: uint32 - function id of result callback.
+    boxHandle: uint32 - encryption box handle. Remark: it can be received through `EncryptionBoxInput` interface.
+
+returns:
+
+    result: uint32 - result code of operation. 0 if succeeded, otherwise SDK error code.
+    decrypted: bytes - decrypted data bytes.
+
+### Signing
+
+`signHash` - sign data hash (256bits) with defined signing box.
+
+arguments: 
+
+    answerId: uint32 - function id of result callback.
+    sbHandle: uint32 - signing box handle.
```

Comment:
Maybe boxHandle to make it more consistent with encryption above?

Reply:
Done